index.js

```diff
@@ -1 +1,92 @@
 // implement your API here
+const express = require('express');
+
+const server = express();
+
+const db = require('./data/db');
+
+server.use(express.json());
+
+server.get('/api', (req, res) => {
+    res.send('Welcome to the Users API!');
+})
+
+server.post('/api/users', (req, res) => {
+    const userInfo = req.body;
+
+    (!userInfo.name || !userInfo.bio)
```

Comment:
Try and use && instead of || here as it will check that both inputs are populated